src/renderer/features/artists/components/album-artist-detail-content.tsx

```diff
@@ -411,6 +417,50 @@ export const AlbumArtistDetailContent = ({ background }: AlbumArtistDetailConten
                         </Group>
                     </Box>
                 ) : null}
+                {externalLinks ? (
+                    <Box
+                        component="section"
+                        mb={-20}
+                        mt={-20}
```

Comment:
Same thing as the AlbumDetailContent comments. Let's normalize the styling to use flex gap between all the sections instead of using inline padding/margin.